homeassistant/components/device_tracker/ee_brightbox.py

```diff
@@ -31,11 +33,64 @@
 })
 
 
-def get_scanner(hass, config):
-    """Return a router scanner instance."""
-    scanner = EEBrightBoxScanner(config[DOMAIN])
-
-    return scanner if scanner.check_config() else None
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up an EE Brightbox device tracker platform."""
+    scanner = EEBrightBoxScanner(config)
+    if not scanner.check_config():
+        return
+    entities = []
+    devices = scanner.scan_devices()
+    for mac in devices:
+        name = scanner.get_device_name(mac)
+        entities.append(EEBrightBoxEntity(mac, name, scanner))
+
+    add_entities(entities, True)
```

Comment:
We need to add new entities when new macs are discovered on the network. Probably use same approach as in owntracks etc.